apiary.apib

```diff
@@ -29,6 +29,9 @@ For example if we have three courses: Intro to Basic Programming, Intro to Advan
 # Issuers
 [Issuers](http://docs.accrediblecredentialapi.apiary.io/#reference/issuer) provide organization details.
 
+# API Key
+API keys are used as Authorization tokens in the header in all authenticated API endpoints. These Keys are managed in the [Accredible Dashboard](https://dashboard.accredible.com/). There are 2 types of API Keys, those that are account wide and work on all assets (Credentials, Groups, Designs etc.) and those that are related to a specific department.
```

Comment:
It sounds like the specific department keys won't work on Credentials, etc with this wording, how about we change it to:

API keys are used as Authorization tokens in the header in all authenticated API endpoints. These Keys are managed in the [Accredible Dashboard](https://dashboard.accredible.com/). There are two types of API Keys, those that are account wide and work on all assets and those that are related to a specific department. API keys scoped to a specific department can only access the resources (Credentials, Groups, Designs etc.) belonging to that department.